crates/oci-distribution/src/lib.rs

```diff
@@ -3,23 +3,18 @@ use chrono::prelude::{DateTime, Utc};
 use futures_util::future;
 use futures_util::stream::StreamExt;
 use hyperx::header::Header;
+use kubelet::{ImageClient, Reference};
```

Comment:
Because we want this to eventually be a fully featured OCI client, I don't think we want to have a dependency on Kubelet in here. `Reference` should most likely be in this crate instead. I'm still thinking through how we'd want to structure things for `ImageClient` though